audio/mididrv.h

```diff
@@ -166,6 +201,13 @@ class MidiDriver : public MidiDriver_BASE {
 	/** Get the device description string matching the given device handle and the given type. */
 	static Common::String getDeviceString(DeviceHandle handle, DeviceStringType type);
 
+	/** Common operations to be done by all drivers on start of send */
+	void sendCommonDriving(uint32 b);
```

Comment:
I would suggest a better name for this, "driving" is not associated with "driver", in this case.
midiDriverCommonSend() sounds better